docs/UserGuide.md

```diff
@@ -400,3 +400,8 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
 | **Note**      | `note INDEX [NOTE]` <br> e.g. <br> `note 2 Sprained right ankle in the past`                                                                                                                        |
 | **Weight**    | `WEIGHT INDEX [WEIGHT]` <br> e.g. <br> `weight 3 70`                                                                                                                                                |
 
+## **Appendix: Planned Enhancements**
+
+**1. Restrict the number of weight entries per day**
+
+Since the variance of a client's weight values in a day is negligible, we can restrict the number of weight entries per day to 1. Having multiple weight entries within a day also distorts the weight-tracking graph.
```

Comment:
```suggestion
Since a client's weight won't change much within a day, we can restrict the number of weight entries per day to 1. Having multiple weight entries within a day also distorts the weight-tracking graph.
```